testing-setup/scala/readme.md

```diff
@@ -0,0 +1,13 @@
+
+## How to start testing in Scala
+
+Scala runs on the JVM, so it requires a JDK. Of course, it also requires a Scala SDK. Setup with Maven isn't _easy_, but once done (this example aims to do this work for you), it should work with Maven commands through a terminal, through JetBrains IntelliJ IDEA (even the Community Edition), Eclipse, or other such tools or IDEs.
+
+1. (Optional) Make a copy of this module, disconnect from the coderetreat remote, or redirect your upstream tracking to your own remote.
```

Comment:
@morganpdx it looks like you have things protected, so perhaps we could delete this step.